### DISCUSSION TITLE
Undefined symbols: gguf_get_val_data

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [ggml-org/llama.cpp#4158] [General]
Hi,

Im getting this error:

```
ld: Undefined symbols:
  _gguf_get_val_data, referenced from:
      gguf_kv_to_str(gguf_context*, int) in llama.cpp.o
```

Anyone knows why?

Im using current master version.

Thanks.

**Comment by KerfuffleV2** [comment]
Try doing `make clean` if you're using a `make` build or whatever the `cmake` equivalent is (I guess just blowing away the build directory). I'd guess your issue is because you have some object files from a previous version laying around.

I think there are some issues with the build system where it doesn't always detect when dependencies need to be rebuild.

**Comment by paulocoutinhox** [comment]
Hi,

It solved o.o

Thanks.
